lib/commands/execute.js

```diff
@@ -79,6 +79,8 @@ extensions.executeMobile = async function (mobileCommand, opts = {}) {
     clearKeychains: 'mobileClearKeychains',
 
     getPermission: 'mobileGetPermission',
+
+    activateSiri: 'mobileActivateSiri',
```

Comment:
I would prefer one of these names:

* `siriCommand`
* `siriVoiceCommand`
* `siri`
* `textToSiri`

Or similar. I just want something that conveys we'll be passing a parameter to get siri to do something. Just saying "activate siri" to me sounds like we're enabling siri on the device, not actually using it.

Reply:
changed to SiriCommand